collections/business/general-notifications/event-general-notifications/batch-status-updated/batch-status-updated/in-app-notification/en_US/direct-debit-engagement-template.json

```diff
@@ -0,0 +1,4 @@
+{
+  "title": "{{#eq event.status 'REJECTED' }}Direct debit batch rejected{{else}}Direct debit payments rejected{{/eq}}",
+  "body": "Batch {{event.name}} of {{event.totalInstructedAmount.currencyCode}} {{formatAmount event.totalInstructedAmount.amount locale}}{{#eq event.status 'REJECTED' }} has been rejected {{else}} has {{event.rejectedTransactionsCount}} rejected {{#eq event.rejectedTransactionsCount 1 }}payment{{else}}payments{{/eq}}{{/eq}}"
+}
```

Comment:
@michalchrust I can see you added 2 templates under the same GN for in app channel. It won't work. You have to create different folders. Please check the [examples](https://github.com/Backbase/engagements-data/tree/master/collections/business/general-notifications/event-general-notifications/audit-export-completed)

Reply:
I changed it as it is in examples you provided. Right now I believe It will work well.